src/execution/operator/persistent/physical_export.cpp

```diff
@@ -36,17 +37,12 @@ static void WriteValueAsSQL(stringstream &ss, Value &val) {
 }
 
 static void WriteCopyStatement(FileSystem &fs, stringstream &ss, TableCatalogEntry *table, CopyInfo &info,
-                               CopyFunction &function) {
-	string table_file_path;
+                               ExportedTableData &exported_table, CopyFunction &function) {
 	ss << "COPY ";
-	if (table->schema->name != DEFAULT_SCHEMA) {
-		table_file_path = fs.JoinPath(
-		    info.file_path, StringUtil::Format("%s.%s.%s", table->schema->name, table->name, function.extension));
-		ss << table->schema->name << ".";
-	} else {
-		table_file_path = fs.JoinPath(info.file_path, StringUtil::Format("%s.%s", table->name, function.extension));
-	}
-	ss << table->name << " FROM '" << table_file_path << "' (";
+
+	ss << KeywordHelper::WriteOptionallyQuoted(exported_table.table_name) << " FROM '" << exported_table.file_path
```

Comment:
I think the table will also need the schema here. Could you add a test case with export in which the table resides in a different schema?

Reply:
It should be taken care already on the bind_export.cpp line 82, this way we avoid having this same `if else` clause at the physical_export.cpp.
```C++
// set up the file name for the COPY TO
if (table->schema->name == DEFAULT_SCHEMA) {
	info->file_path =
			  fs.JoinPath(stmt.info->file_path,
			              StringUtil::Format("%s_%s.%s", to_string(id), SanitizeExportIdentifier(table->name),
			                                 copy_function->function.extension));
} else {
	info->file_path = fs.JoinPath(
			  stmt.info->file_path,
			  StringUtil::Format("%s_%s_%s.%s", SanitizeExportIdentifier(table->schema->name), to_string(id),
			                     SanitizeExportIdentifier(table->name), copy_function->function.extension));
}
```

In any case, I will add the tests because my reasoning might be wrong :)

Reply:
My reasoning was wrong